vite.config.ts

```diff
@@ -1,8 +1,19 @@
 import { defineConfig } from 'vite';
+import { viteStaticCopy } from 'vite-plugin-static-copy';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  build: {
-    sourcemap: true,
-  },
+	build: {
+		sourcemap: true,
+	},
+	plugins: [
+		viteStaticCopy({
```

Comment:
Is there a particular reason this doesn't work out of the box in the normal "public" folder?